samples/04_gis_analysts_data_scientists/classifying_human_activity_using _tabPFN_classifier.ipynb

```diff
@@ -0,0 +1,2179 @@
+{
```

Comment:
<p>Next, we will evaluate the model's performance. This will print out multiple model metrics that we can use to assess the model quality. These metrics include a combination of multiple evaluation criteria, such as&nbsp;<code>accuracy</code>,&nbsp;<code>precision</code>,&nbsp;<code>recall</code>&nbsp;and&nbsp;<code>F1-Score</code>, which collectively measure the model's performance on the validation set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>